### ISSUE TITLE
Bug: No generate models

### ISSUE THREAD

**Issue by Appjak34** [YassinEldeeb/prisma-tgql-types-gen#25]
## Bug description
I set my project as your readme, but it does not generate the models when I run the command "npm prisma generate"

## Screenshots

<img width="654" alt="Captura de Pantalla 2021-12-11 a la(s) 10 14 48" src="https://user-images.githubusercontent.com/23286030/145683618-b0a59652-f7d7-4a5e-b58d-d258fd04e23a.png">

## Prisma information
```prisma
generator client {
  provider = "prisma-client-js"
}

generator PrismaTypeGraphQLTypesGenerator {
  provider     = "npx prisma-typegraphql-types-generator"
  modelsOutput = "./src/models"
  enumsOutput  = "./src/types/enums"
  useYarn      = true
}

datasource db {
  provider = "postgresql"
  url      = env("DATABASE_URL")
}

model User {
  id                            String                 @id @default(cuid())
  name                          String                 @db.VarChar(255)
  // @nullable
  email                         String                 @unique
  // @skip
  password                      String?
  bio                           String?                @db.VarChar(160)
  // @skip
  tokenVersion                  Int                    @default(0)
  // @skip
  confirmed                     Boolean                @default(false)
  profilePic                    String?
  posts                         Post[]
  githubId                      String?                @unique
  lastTimelineVisit             DateTime?
  createdAt                     DateTime               @default(now())
  updatedAt                     DateTime               @updatedAt
}

model Post {
  id             String    @id @default(cuid())
  title          String
  body           String
  tags           String[]
  published      Boolean   @default(false)
  // @skip
  authorId       String
  author         User      @relation(fields: [authorId], references: [id], onDelete: Cascade)
  readingTimeTxt String
  readingTimeMin Float
  coverImg       String?
  createdAt      DateTime  @default(now())
  updatedAt      DateTime  @updatedAt
}
```

## Prisma  Version
```
prisma                  : 3.6.0
@prisma/client          : 3.6.0
Current platform        : darwin
Query Engine (Node-API) : libquery-engine dc520b92b1ebb2d28dc3161f9f82e875bd35d727 (at node_modules/@prisma/engines/libquery_engine-darwin.dylib.node)
Migration Engine        : migration-engine-cli dc520b92b1ebb2d28dc3161f9f82e875bd35d727 (at node_modules/@prisma/engines/migration-engine-darwin)
Introspection Engine    : introspection-core dc520b92b1ebb2d28dc3161f9f82e875bd35d727 (at node_modules/@prisma/engines/introspection-engine-darwin)
Format Binary           : prisma-fmt dc520b92b1ebb2d28dc3161f9f82e875bd35d727 (at node_modules/@prisma/engines/prisma-fmt-darwin)
Default Engines Hash    : dc520b92b1ebb2d28dc3161f9f82e875bd35d727
Studio                  : 0.440.0
```

**Comment by YassinEldeeb** [comment]
This looks good to me, where is the problem?
you should have your generated models at `./src/generated/models`

**Comment by Appjak34** [comment]
The folders were created empty, without the class files.


#Environment & setup
OS: MacOs BigSur 11.4
Database: postgresql
Node.js version: v14.17.0

```
api
├─ .dockerignore
├─ .editorconfig
├─ .git
│  ├─ HEAD
│  ├─ config
│  ├─ description
│  ├─ hooks
│  │  ├─ applypatch-msg.sample
│  │  ├─ commit-msg.sample
│  │  ├─ fsmonitor-watchman.sample
│  │  ├─ post-update.sample
│  │  ├─ pre-applypatch.sample
│  │  ├─ pre-commit.sample
│  │  ├─ pre-merge-commit.sample
│  │  ├─ pre-push.sample
│  │  ├─ pre-rebase.sample
│  │  ├─ pre-receive.sample
│  │  ├─ prepare-commit-msg.sample
│  │  ├─ push-to-checkout.sample
│  │  └─ update.sample
│  ├─ index
│  ├─ info
│  │  └─ exclude
│  ├─ objects
│  │  ├─ 7d
│  │  │  └─ b7fd00077007c40b9244dd767729cf8c6790ef
│  │  ├─ e6
│  │  │  └─ 9de29bb2d1d6434b8b29ae775ad8c2e48c5391
│  │  ├─ info
│  │  └─ pack
│  │     ├─ pack-a4583b9c20762fede0bc74df866d4beea998384e.idx
│  │     └─ pack-a4583b9c20762fede0bc74df866d4beea998384e.pack
│  ├─ packed-refs
│  └─ refs
│     ├─ heads
│     │  └─ main
│     ├─ remotes
│     │  └─ origin
│     │     └─ HEAD
│     └─ tags
├─ .gitattributes
├─ .gitignore
├─ .idea
│  ├─ .gitignore
│  ├─ api.iml
│  ├─ misc.xml
│  ├─ modules.xml
│  ├─ vcs.xml
│  └─ workspace.xml
├─ Dockerfile
├─ LICENSE
├─ README.md
├─ docker-compose.yml
├─ globalConfig.json
├─ jest-mongodb-config.js
├─ jest.config.js
├─ models
├─ package-lock.json
├─ package.json
├─ prisma
│  └─ schema.prisma
├─ src
│  ├─ bootstrap
│  │  ├─ apollo.ts
│  │  ├─ express.ts
│  │  ├─ index.ts
│  │  └─ prisma.ts
│  ├─ config.ts
│  ├─ entities
│  │  ├─ index.ts
│  │  └─ todo.ts
│  ├─ generated
│  │  └─ models
│  ├─ index.ts
│  ├─ models
│  ├─ modules
│  │  ├─ index.ts
│  │  └─ todo
│  │     ├─ input.ts
│  │     ├─ model.ts
│  │     ├─ resolver.ts
│  │     └─ service.ts
│  ├─ test
│  │  ├─ integration
│  │  │  ├─ __snapshots__
│  │  │  │  └─ todo.test.ts.snap
│  │  │  └─ todo.test.ts
│  │  └─ utils
│  │     └─ index.ts
│  ├─ types
│  └─ utils
│     ├─ buildSchema.ts
│     ├─ index.ts
│     └─ scalars.ts
└─ tsconfig.json

```

**Comment by YassinEldeeb** [comment]
That's pretty strange.
I used the same `Environment & setup` as yours except for the the `OS: MacOs BigSur 11.4` and It works pretty well.

**Comment by YassinEldeeb** [comment]
hey that was a tough one, found the issue and fixed it with `v1.5.10`
